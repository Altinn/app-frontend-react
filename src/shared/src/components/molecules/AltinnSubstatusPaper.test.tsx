import * as React from 'react';
import { render, screen } from '@testing-library/react';
import AltinnSubstatusPaper from './AltinnSubstatusPaper';
import 'jest';

describe('AltinnSubstatusPaper', () => {
  const label = 'The label';
  const description = 'The description';
  it('should match snapshot', () => {
    const { container } = render(
      <AltinnSubstatusPaper label={label} description={description} />,
    );
    expect(container.firstChild).toMatchSnapshot();
  });

  it('should render label and description', () => {
    render(<AltinnSubstatusPaper label={label} description={description} />);
    expect(screen.getByText(/the label/i)).toBeInTheDocument();
    expect(screen.getByText(/the description/i)).toBeInTheDocument();
  });
});
