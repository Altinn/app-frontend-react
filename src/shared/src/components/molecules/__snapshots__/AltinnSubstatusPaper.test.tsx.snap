// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AltinnSubstatusPaper should match snapshot 1`] = `
<div
  class="MuiPaper-root makeStyles-paper-1 MuiPaper-elevation0 MuiPaper-rounded"
>
  <div
    class="MuiGrid-root MuiGrid-container MuiGrid-direction-xs-column"
  >
    <div
      class="MuiGrid-root MuiGrid-item"
    >
      <p
        class="MuiTypography-root MuiTypography-body1"
        id="substatus-label"
        style="font-size: 2.4rem; margin-bottom: 0.8rem;"
      >
        The label
      </p>
    </div>
    <div
      class="MuiGrid-root MuiGrid-item"
    >
      <p
        class="MuiTypography-root MuiTypography-body1"
        id="substatus-description"
        style="font-size: 1.8rem;"
      >
        The description
      </p>
    </div>
  </div>
</div>
`;
